modules/reindex/src/main/java/org/opensearch/index/reindex/ReindexValidator.java

```diff
@@ -101,7 +101,7 @@ static void checkRemoteWhitelist(CharacterRunAutomaton whitelist, RemoteInfo rem
         if (whitelist.run(check)) {
             return;
         }
-        String whiteListKey = TransportReindexAction.REMOTE_CLUSTER_WHITELIST.getKey();
+        String whiteListKey = TransportReindexAction.REMOTE_CLUSTER_ALLOWLIST.getKey();
         throw new IllegalArgumentException('[' + check + "] not whitelisted in " + whiteListKey);
```

Comment:
Can we change this exception message?

Reply:
Hi @andrross, renaming the exception message is covered in PR [#2178 ](https://github.com/opensearch-project/OpenSearch/pull/2178/files#diff-8b7f09110a1fca14cdf4e4c35a427217cfe86d825451e56ad2f30b75ed391d33), so I didn't touch it. 😁

Reply:
Since PR #2178 is merged, the "whitelisted" in the exception messages have changed to "allowlisted".